docs/quickstart/quickstart_chains/terra.md

```diff
@@ -29,7 +29,7 @@ type Transfer @entity {
 }
 ```
 
-::: warning Important
+::: info Note
```

Comment:
What if we use info Important here? `info` as the container and `Important as the text`. This step is important for users and they must not skip it. And since we don't want users to bombard with unnecessary warnings, should we give it a try to this combination in this case?

Reply:
@ZeelPatel1102 I reversed it back to `warning` containers. It is not a note, you are right, this is crucial step. I think `warning` containers fits here perfectly - users will benefit from noticing this, for sure. 

Also, I wouldn't use `info` + _Important_; just to keep it all coherent, but I am open to discussing it, of course :)